src/platform/api/pythonApi.ts

```diff
@@ -435,9 +435,6 @@ export class InterpreterService implements IInterpreterService {
             this.disposables
         );
     }
-    public async waitForAllInterpretersToLoad(): Promise<void> {
```

Comment:
removed this method and ensure we call refresh, thats the same
this way we have one less method